content/200-concepts/100-components/01-prisma-schema/06-relations/300-many-to-many-relations.mdx

```diff
@@ -413,28 +413,40 @@ When you don't need to attach additional information to the relation, you can mo
 
 ## MongoDB
 
-In MongoDB, m-n-relations are represented by:
+In MongoDB, many-to-many (M:N) relations are represented by:
 
-- Relation fields on both sides, include a `@relation` attribute on _each side_
+- Relation fields on both sides, include a `@relation` attribute on _each side_, with mandatory `fields` and `references` arguments
 - A scalar list of referenced IDs on each side, including the `@db.ObjectId` attribute
 
-The following example demonstrate an m-n relation between posts and categories:
+The following example demonstrate an M:N relation between posts and categories:
 
 ```prisma
 model Post {
   id          String     @id @default(auto()) @map("_id") @db.ObjectId
   categoryIDs String[]   @db.ObjectId
-  categories  Category[] @relation(fields: [categoryIDs])
+  categories  Category[] @relation(fields: [category_ids], references: [id])
 }
 
 model Category {
   id      String   @id @default(auto()) @map("_id") @db.ObjectId
   postIDs String[] @db.ObjectId
-  posts   Post[]   @relation(fields: [postIDs])
+  posts   Post[]   @relation(fields: [post_ids], references: [id])
 }
 ```
 
-### Querying MongoDB m-n relations
+### Validation rules (Preview)
+
+For MongoDB, Prisma validates M:N relations with the following rules:
+
+- Both sides of the relation must be arrays (in the data model the fields categories and posts).
```

Comment:
Do we call these arrays or lists usually? I think lists.

Also:
```suggestion
- Both sides of the relation must be lists (in the example data model above the fields `categories` and `posts`).
```

Reply:
@pimeys can you confirm... shall we use the term "lists" or "arrays"?

Reply:
Looks like we're using 'list' normally [prisma.io/docs/reference/api-reference/prisma-schema-reference#-modifier](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#-modifier)